infrastructure/main.tf

```diff
@@ -68,6 +68,11 @@ module "app" {
     # Toggles
     ENABLE_IDAM_HEALTH_CHECK = "${var.enable_idam_healthcheck}"
     ENABLE_S2S_HEALTH_CHECK = "${var.enable_s2s_healthcheck}"
+
+    HTTP_SCHEME = "https"
+    HTTP_HOST_PORT = "443"
+
+    CCD_DATA_API_URL = "http://${var.ccd_data_store_api_url}-${local.local_env}.service.core-compute-${local.local_env}.internal"
```

Comment:
why specify a http scheme and then not use it?

Reply:
https is used to build callbacks